docs/migration_guide.rst

```diff
@@ -9,32 +9,44 @@ Version 0.19.0 of **starknet.py** comes with support for RPC 0.6.0!
 
 .. currentmodule:: starknet_py.net.client_models
 
-New classes added to mirror the recent changes in the RPC v0.6.0 specification include:
+New classes added to mirror the recent changes in the RPC v0.6.0 specification that include:
 :class:`ResourceBoundsMapping`, :class:`ResourceBounds`, :class:`PriceUnit`, :class:`FeePayment`, :class:`DAMode`.
 
 Changes in the :class:`~starknet_py.net.account.account.Account`:
 
 .. currentmodule:: starknet_py.net.account.account
 
+- :meth:`~Account.execute` has been renamed to :meth:`~Account.execute_v1`
 - :meth:`~Account.execute_v3` has been added.
-- :meth:`~Account.sign_declare_v3_transaction`, :meth:`~Account.sign_deploy_account_v3_transaction` and :meth:`~Account.sign_invoke_v3_transaction` have been added.
-- :meth:`~Account.sign_declare_transaction`, :meth:`~Account.sign_deploy_account_transaction` and :meth:`~Account.sign_invoke_transaction` have been renamed to :meth:`~Account.sign_declare_v1_transaction`, :meth:`~Account.sign_deploy_account_v1_transaction` and :meth:`~Account.sign_invoke_v1_transaction` respectively.
+- :meth:`~Account.deploy_account` has been renamed to :meth:`~Account.deploy_account_v1`
+- :meth:`~Account.deploy_account_v3` has been added
+- :meth:`~Account.sign_declare_v3_transaction`, :meth:`~Account.sign_deploy_account_v3_transaction` and :meth:`~Account.sign_invoke_v3_transaction` have been added
```

Comment:
Given that we already have all these breaking changes, what do you think about removing the word `transaction` from the `Account` methods? This would result in names like `sign_invoke_v1`, `sign_declare_v3`, etc.

Reply:
I'm fine with that